src/backend/joanie/tests/signature/test_backend_signature_get_backend.py

```diff
@@ -0,0 +1,90 @@
+"""get_signature_backend() test suite"""
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from joanie.signature.backends import get_signature_backend
+
+
+class GetSignatureBackendTestSuite(TestCase):
+    """Test suite for the get_signature_backend method."""
+
+    @override_settings(
+        JOANIE_SIGNATURE_BACKEND="joanie.signature.backends.base.BaseSignatureBackend"
+    )
+    def test_backend_get_signature_get_backend_base(self):
+        """
+        When JOANIE_SIGNATURE_BACKEND is well configured in settings, the base signature backend
+        is returned when calling the method `get_signature_backend`
+        """
+
+        backend = get_signature_backend()
+
+        self.assertEqual(backend.name, "base")
```

Comment:
To be sure, you can test here the instance type. And do it in every tests in this file.